server/interfaces.coffee

```diff
@@ -32,4 +35,25 @@ Meteor.methods
         offset = UTCOffsets[tz]
         dateUTC = match[1].replace(' ', 'T') + offset
         dateUTC
-    )
+
+  queryForSuggestedArticles: (eventId) ->
+    check eventId, String
+    event = grid.UserEvents.findOne(eventId)
+    console.log "Calling SPA API @ " + SPA_API_URL
+    unless event
+      throw new Meteor.Error 404, "Unable to fetch the requested event record"
+    keywords = _.uniq event.eventName.match(/\w{3,}/g)
+    mongoQuery = { $and: [] }
+    for keyword in keywords
```

Comment:
I added a text search endpoint since it will be faster and allow sorting by text similarity: https://github.com/ecohealthalliance/spa/blob/master/server/api.coffee#L4
Could you use that instead? (The /find endpoint will be useful for other things like pulling new articles into the curator inbox.)